Refactoring and styling of layer controls

atmeta opened the pull request:
New and pretty layer controls (use existing leaflet classes, and no id).

Also (!) 
* added the .js.map-files that made the 404 error in console on /events/map
* block extra_css in base.html template is moved so the specificity applies better to our primordialsoup.css

atmeta committed:
Merge branch 'development' of github.com:ursuppeeppusru/ursuppe into map-layers

atmeta committed:
Add parent and sub layer for all and current events respectively

atmeta committed:
Add leaflet .js and .css files to commit

atmeta committed:
Change label closing soon to only be on events that also are current

atmeta committed:
Add zoom when marker is clicked, on event details map

atmeta committed:
Add zoom out when marker is clicked when zoomed in

atmeta committed:
Add toggle button to map

atmeta committed:
Merge branch 'development' of github.com:ursuppeeppusru/ursuppe into map-layers

atmeta committed:
Fixed exhibitions label 'closing soon' to only be on current exhibitions in event details and highlights

atmeta committed:
Add leaflet.css and .js

atmeta committed:
Add custom checkboxes

atmeta committed:
Styling of layer control incl. checkboxes

atmeta committed:
Resolved merge conflict

atmeta committed:
Adjusted map sleep time to 250ms

atmeta committed:
Title in django-admin refers to .env-file

adionditsak commented:
![Screenshot 2024-05-23 at 13 31 26](https://github.com/ursuppeeppusru/ursuppe/assets/1625366/60628060-1aca-4631-b112-0cc88856e958)

When i test locally the map is empty. Can you reproduce this error?

atmeta commented:
Fixed. I just pushed again @adionditsak :) 

Also added..
* map will sleep faster when cursor is away from map.
* in /django-admin site name refers to settings/.env.

atmeta committed:
Event detail map height same as main map

atmeta committed:
Merge branch 'development' of github.com:ursuppeeppusru/ursuppe into map-layers

atmeta committed:
Tidy .css of empty declaration blocks and invalid syntax as pointed out by VSCode

adionditsak commented on primordialsoup/static/css/primordialsoup.css:
```diff
@@ -65,6 +65,12 @@ body {
 }
 
+@font-face {
+    font-family: basteleurbold;
```
Tror bare den her skal hedde det samme som den anden `basteleur`, så bruger den automatisk bold versionen her hvis man bruger font-weight bold eller <b>

atmeta replied:
Nej -det er to forskellige (Moonlight og Bold). Jeg bruger den til '+' og '-' i zoom knapperne til kortet. Se: https://velvetyne.fr/fonts/basteleur/

atmeta replied:
<img width="280" alt="Screenshot 2024-05-23 at 14 56 01" src="https://github.com/ursuppeeppusru/ursuppe/assets/43446790/2a24132f-14ec-43ab-98b8-981860e78634">

adionditsak replied:
Ah okay, i princippet en helt anden font så? Normalt er der jo en default og f.eks. en bold, hvor man ikke vil bruge bold i dens font-family definition i font-face.

atmeta replied:
New push :)

adionditsak commented on ps_calendar/templates/event_submission_map.html:
```diff
@@ -25,11 +25,7 @@ <h5><b>EVENTS</b></h5>
-        <button id="toggle-map-layers">Show only current events</button>
-      </div>      
-</div>
+<div id="map"></div>
```
Kæmpe forbedring med kortet! Godt gået!

atmeta committed:
BasteleurBold now secondary header font
